batch/batch/driver/main.py

```diff
@@ -552,6 +559,27 @@ def validate_int(session, name, value, predicate, description):
     return validate(session, name, i, predicate, description)
 
 
+@routes.post('/configure-feature-switches')
+@check_csrf_token
+@auth.web_authenticated_developers_only()
+async def configure_feature_switches(request, userdata):  # pylint: disable=unused-argument
+    app = request.app
+    db: Database = app['db']
+    post = await request.post()
+
+    compact_billing_tables = 'compact_billing_tables' in post
+
+    await db.execute_update(
+        '''
+UPDATE feature_switches SET compact_billing_tables = %s;
```

Comment:
Let's consistently use the same name everywhere. `feature_flags` is the industry standard term for this functionality.